tests/cephfs/nfs/data_backup_restore_bw_nfs_and_local_mounts.py

```diff
@@ -0,0 +1,164 @@
+        client1 = clients[0]
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+        rhbuild = config.get("rhbuild")
+        nfs_servers = ceph_cluster.get_ceph_objects("nfs")
+        nfs_server = nfs_servers[0].node.hostname
+        nfs_name = "cephfs-nfs"
+        client1.exec_command(sudo=True, cmd="ceph mgr module enable nfs")
+        client1.exec_command(
+            sudo=True, cmd=f"ceph nfs cluster create {nfs_name} {nfs_server}"
+        )
+        if wait_for_process(client=client1, process_name=nfs_name, ispresent=True):
+            log.info("ceph nfs cluster created successfully")
+        else:
+            raise CommandFailed("Failed to create nfs cluster")
+        nfs_export_name = "/export_" + "".join(
+            secrets.choice(string.digits) for i in range(3)
+        )
+        export_path = "/"
+        fs_name = "cephfs"
+        if "5.0" in rhbuild:
+            client1.exec_command(
+                sudo=True,
+                cmd=f"ceph nfs export create cephfs {fs_name} {nfs_name} "
+                f"{nfs_export_name} path={export_path}",
+            )
+        else:
+            client1.exec_command(
+                sudo=True,
+                cmd=f"ceph nfs export create cephfs {nfs_name} "
+                f"{nfs_export_name} {fs_name} path={export_path}",
+            )
+        out, rc = client1.exec_command(sudo=True, cmd=f"ceph nfs export ls {nfs_name}")
+
+        if nfs_export_name not in out:
+            raise CommandFailed("Failed to create nfs export")
+
+        log.info("ceph nfs export created successfully")
+        out, rc = client1.exec_command(
+            sudo=True, cmd=f"ceph nfs export get {nfs_name} {nfs_export_name}"
+        )
+        output = json.loads(out)
+        mounting_dir = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(10))
+        )
+        nfs_mounting_dir = f"/mnt/cephfs_nfs{mounting_dir}_1/"
+        client1.exec_command(sudo=True, cmd=f"mkdir -p {nfs_mounting_dir}")
+        command = f"mount -t nfs -o port=2049 {nfs_server}:{nfs_export_name} {nfs_mounting_dir}"
+        output, err = client1.exec_command(sudo=True, cmd=command, check_ec=False)
+
+        local_dir = f"/mnt/local_{mounting_dir}_1/"
+        client1.exec_command(sudo=True, cmd=f"mkdir -p {local_dir}")
+        run_ios(
+            clients[0], nfs_mounting_dir, file_name="dd_file_nfs1", bs="100M", count=20
+        )
+        run_ios(
+            clients[0], nfs_mounting_dir, file_name="dd_file_nfs2", bs="100M", count=20
+        )
+
+        log.info("Backup the data from NFS mounts to local dir")
+        nfspath1 = f"{nfs_mounting_dir}{clients[0].node.hostname}dd_file_nfs1"
+        nfspath2 = f"{nfs_mounting_dir}{clients[0].node.hostname}dd_file_nfs2"
+        commands = [
+            f"rsync {nfspath1} {local_dir}",
+            f"scp {nfspath2} {local_dir}",
```

Comment:
Could we also add remote copying to other clients from nfs mount using rsync & scp & vice-versa

Reply:
I think there is another TC which has this remote copy.